src/bci_build/package/tomcat/README.md.j2

```diff
@@ -0,0 +1,82 @@
+# Tomcat {{ image.version }} Container Image
+{% include 'badges.j2' %}
+
+Apache Tomcat (Tomcat for short) is a free and open-source implementation of the
+Jakarta Servlet, Jakarta Expression Language, and WebSocket technologies. It
+provides a pure Java HTTP web server environment that can run Java code. It is a
+Java web application server and not a complete JEE application server.
+
+
+## How to use the image
+
+By default, the image launches Tomcat with the same configuration as the one
+that comes with SUSE Linux Enterprise Server. The difference is that logging is
+sent to stdout, meaning that the `podman logs tomcat` command displays Tomcat
+logs.
+
+For security reasons, the image runs as the **tomcat** user. This means that
+additional packages cannot be installed via `zypper`, unless the user becomes
+`root`.
+
+To deploy an application, copy the `.war` file file into
+`$CATALINA_BASE/webapps` (either during a container build or by bind-mounting
+the directory), and launch the container using the following command:
+```ShellSession
+$ podman run -d --rm -p 8080:8080 {{ image.pretty_reference }}
+```
+
+The deployed webapp is then accessible via `http://localhost:8080/$webapp_name`.
+
+
+### How to use the image with rootless Podman
+
+The container image can be used in rootless mode with Podman. Keep in mind that
+Podman remaps the `tomcat` user in the container to a different user on the
+host. This user will not have write access to the mounted directory. To avoid
+permission issues change permissions of the shared directory to `0777` as
+follows:
+
+```ShellSession
+$ chmod 0777 /path/to/my/app
+$ podman run --rm -d -v /path/to/my/app:{{ image.env['CATALINA_HOME'] }}/webapps:z \
+      -p 8080:8080 {{ image.pretty_reference }}
+```
+
+
+## Configuration
+
+The main Tomcat configuration files (for example
+`/etc/tomcat/logging.properties`) are stored in `/etc/tomcat/`.
+
+Tomcat's runtime options can be configured using the environment variables
+`JAVA_OPTS` and `CATALINA_OPTS`. `JAVA_OPTS` specifies general options used for
+the JVM, whereas `CATALINA_OPTS` specifies Tomcat's flags. You can pass the
+options to the container runtime using the `-e` flag:
+```ShellSession
+$ podman run -it --rm \
+      -e JAVA_OPTS="-Xmx1024m" -p 8080:8080 \
+      {{ image.pretty_reference }}
+```
+
+The image ships with `CATALINA_HOME` set to `{{ image.env['CATALINA_HOME'] }}`
+and `CATALINA_BASE` set to `{{ image.env['CATALINA_BASE'] }}`.
+
+
+## Samples
+
+By default, the sample applications shipped with Tomcat are not installed in
+the container image. Add them by installing one of the following
+packages:
+- {{ image.package_list[0] }}-webapps
+- {{ image.package_list[0] }}-admin-webapps
+- {{ image.package_list[0] }}-docs-webapp
+
```

Comment:
Actually, this whole section needs to go anyway, as the samples are not shipped in the repo
```suggestion
```

Reply:
the ones I listed are shipped in the repo: 

```
S | Name                   | Type    | Version               | Arch   | Repository
--+------------------------+---------+-----------------------+--------+-----------
  | tomcat-admin-webapps   | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat-docs-webapp     | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat-webapps         | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat10-admin-webapps | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
  | tomcat10-docs-webapp   | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
  | tomcat10-webapps       | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
```

Reply:
also, please apply your tomcat related review to the #1073  PR where this commit originates from. this one is supposed to come after #1073 is reviewed and merged.

Reply:
Well, you've merged #1073 without waiting for my review there :shrug:

I just recall that you said that the `*-webapps` packages are in the webscripting module and should not be in SLE_BCI. I explicitly added them to the SLE_BCI repo, hence why they are available.

Reply:
@dcermak it was just moving code around, we don't need to have two reviews for such simple changes. An isolated PR for this matter would have been sufficient. 

Anyhow, I do remember that `*-docs-webapp` should have been removed. I'll send a PR for that.

Reply:
moved to #1084